eng/versioning/external_dependencies.txt

```diff
@@ -54,9 +54,12 @@ org.slf4j:slf4j-simple;1.7.25
 cglib:cglib-nodep;3.2.7
 com.github.tomakehurst:wiremock-standalone;2.24.1
 com.microsoft.azure:adal4j;1.6.5
+com.microsoft.azure:azure;1.24.1
 com.microsoft.azure:azure-mgmt-graph-rbac;1.3.0
 com.microsoft.azure:azure-mgmt-keyvault;1.11.1
+com.microsoft.azure:azure-mgmt-network:1.33.0
```

Comment:
@bgklein blarg, I'm sorry, there was a typo in the feedback, the last colon needs to be a semicolon. com.microsoft.azure:azure-mgmt-network:1.33.0 should be
com.microsoft.azure:azure-mgmt-network;1.33.0